source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -275,6 +392,11 @@ def _build_fallback(
     fallback_logic = _get_fallback_logic(fallback_type)
 
     pt_obj = fallback_logic.get_pt_boundaries(pt_journey)
+    car_park = None
+    car_park_crowfly_duration = None
+
+    if mode == 'car':
+        _, _, car_park, car_park_crowfly_duration = fallback_durations_pool.wait_and_get(mode)[pt_obj.uri]
```

Comment:
`fallback_durations_pool.wait_and_get(mode)` is already call here https://github.com/CanalTP/navitia/pull/3213/files#diff-9ca4efa8d09d46bc612e4aa3588368b4R391
You can save on call